### DISCUSSION TITLE
Age restrictions in ytdl-core did not overcome by using "Cookie" header

### DISCUSSION THREAD

**Discussion by danyaridiger** [Androz2091/discord-player#1839] [Q&A] [answered]
Hi, there! I tryed to load video from Youtube in my app using discord-player with ytdl-core config options which looks like: 
`
client.player = new Player(client, {
  ytdlOptions: {
    filter: "audioonly",
    quality: 'highestaudio', 
    highWaterMark: 1 << 25,
    requestOptions: {
      headers: {
        cookie: cookie.toString(),
      },
    },
  },
});
`

When `cookie.toString()` is a buffer from text file with cookie (i checked it for correct parsing)

But i have got an error "ERR_NO_RESULT: Could not extract stream for this track. Error: Sign in to confirm your age". 

How to correctly paste cookie in discord player configuration?

**Comment by twlite** [comment, marked as answer]
> Hi, there! I tryed to load video from Youtube in my app using discord-player with ytdl-core config options which looks like: `client.player = new Player(client, { ytdlOptions: { filter: "audioonly", quality: 'highestaudio', highWaterMark: 1 << 25, requestOptions: { headers: { cookie: cookie.toString(), }, }, }, });`
> 
> When `cookie.toString()` is a buffer from text file with cookie (i checked it for correct parsing)
> 
> But i have got an error "ERR_NO_RESULT: Could not extract stream for this track. Error: Sign in to confirm your age".
> 
> How to correctly paste cookie in discord player configuration?

your cookie may have expired, can you try again with new cookie? Also what are you using as ytdl engine? Engines such as `yt-stream` do not accept cookie header.

**Reply by danyaridiger** [reply]
I use `ytdl-core` engine under the hood. Previously this config always worked correctly, but now i can assume that something wrong in this one indeed, because cookie were suspended before i have started this discussion(
